docs/get-started/whats-a-story.md

```diff
@@ -10,6 +10,17 @@ Each example component has a set of stories that show the states it supports. Yo
 
 Let’s start with the `Button` component. A story is a function that describes how to render the component in question. Here’s how to render `Button` in the “primary” state and export a story called `Primary`.
 
+<details>
+<summary><h4 id="troubleshooting">Additional steps for using Native Format</h4></summary>
+
+- Install [addon-svelte-csf](https://github.com/storybookjs/addon-svelte-csf) as a dependency.
+- Update `.storybook/main.js` to include "@storybook/addon-svelte-csf" in the addons array.
+- Update `.storybook/main.js` to set `storyStoreV7` to false.
+
+If all else fails, try asking for [help](https://storybook.js.org/support)
+
+</details>
```

Comment:
@octoxan, to avoid introducing confusion to our users and if you're ok with it, the change you're introducing should live in this [file](https://github.com/storybookjs/storybook/blob/next/docs/get-started/installation-problems/svelte.mdx). We adjust the bullet point regarding Svelte Native format for clarity based on what you have here. Does it sound good to you?